build.gradle

```diff
@@ -12,7 +12,7 @@ repositories {
 
 
 dependencies {
-	compile 'net.java.dev.jna:jna:3.5.2'
+	compile 'net.java.dev.jna:jna:4.1.0'
```

Comment:
Please don't change this, it causes issues. https://github.com/EtiennePerot/fuse-jna/issues/28